package.json

```diff
@@ -41,6 +41,7 @@
     "react": "^15.4.1"
   },
   "dependencies": {
-    "autobind-decorator": "^1.3.4"
+    "autobind-decorator": "^1.3.4",
+    "prop-types": "^15.6.0"
```

Comment:
We should also test it with react 16, react and prop-types deps should have the same version.